src/entities/MovieTitle/MovieTitle.jsx

```diff
@@ -0,0 +1,48 @@
+import "./MovieTitle.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEllipsisVertical } from "@fortawesome/free-solid-svg-icons";
+import { useState } from "react";
+import cx from "classnames";
+
+export const MovieTitle = ({ movieTitle }) => {
+  const { imgUrl, movieName, releaseYear, genres, onClick } = movieTitle;
+  const [isContextDropdownOpen, setContextDropdownOpen] = useState(false);
+
+  const toggleDropdown = () => {
+    setContextDropdownOpen((prevValue) => !prevValue);
+  };
+
+  return (
+    <div className="movie-title" onClick={onClick}>
+      <img src={imgUrl} alt="movie" className="movie-image" />
+      <div className="context-menu">
+        <FontAwesomeIcon
+          icon={faEllipsisVertical}
+          className="context-icon"
+          onClick={toggleDropdown}
+        />
+        <div
+          className={cx("items", {
+            active: isContextDropdownOpen,
+            hidden: !isContextDropdownOpen
+          })}
```

Comment:
what about using `active` styles as default and using `hidden` conditionally?
```suggestion
          className={cx("items", {
            hidden: !isContextDropdownOpen
          })}
```

Reply:
done